src/patternfly/components/FormControl/form-control.scss

```diff
@@ -60,6 +64,17 @@ $pf-c-form-control__select--ViewBox: "320" !default;
   --pf-c-form-control--invalid--exclamation--Background: var(--pf-c-form-control--invalid--BackgroundUrl) var(--pf-c-form-control--invalid--BackgroundPosition) / var(--pf-c-form-control--invalid--BackgroundSize) no-repeat;
   --pf-c-form-control--invalid--Background: var(--pf-c-form-control--BackgroundColor) var(--pf-c-form-control--invalid--exclamation--Background);
 
+  // Input m-validated
+  --pf-c-form-control--validated--BorderBottomWidth: var(--pf-global--BorderWidth--md);
+  --pf-c-form-control--validated--PaddingBottom: calc(var(--pf-global--spacer--form-element) - var(--pf-c-form-control--validated--BorderBottomWidth));
+  --pf-c-form-control--validated--BorderBottomColor: var(--pf-global--success-color--100);
+  --pf-c-form-control--validated--PaddingRight: var(--pf-global--spacer--xl);
+  --pf-c-form-control--validated--BackgroundPosition: calc(100% - var(--pf-c-form-control--PaddingLeft)) var(--pf-c-form-control--PaddingLeft);
```

Comment:
I wonder if we could group the BackgroundPosition and BackgroundURL that is set on the validated/invalid modifier to `pf-c-form-control` since its used in both places.

Reply:
I could go either way with dropping `BackgroundPosition` - both the default (for input and textarea), as well as the ones for the select variation. It is duplicated, but that also allows for flexibility in case we were to use different icons  and wanted different spacing. 

I think we want to keep BackgroundURL unique between the default select, invalid and validated since the function that generates the value takes different arguments between the 3.

Reply:
Just let me know what we want here.

Reply:
It's just an extra variable and allows for more flexibility. I think this is fine to leave as-is unless you disagree @christiemolloy.

Reply:
yep thats ok with me